include/zephyr/dt-bindings/gpio/nordic-nrf-gpio.h

```diff
@@ -1,74 +1,64 @@
 /*
- * Copyright (c) 2022 Vestas Wind Systems A/S
+ * Copyright (c) 2022 Nordic Semiconductor ASA
```

Comment:
Please do not remove copyright statements. Just add an additional line.

Reply:
file has been re-written from scratch, there's no re-used content

Reply:
I think this is a slippery slope. If we go down this road we can start discussing what constitutes a total rewrite in all PRs and whether original copyrights should be kept or not. 

Say you changed half the file in one PR and the other half in a second PR. Should the second PR then remove all copyrights than yours?

Please leave existing copyrights in place.

Reply:
I agree with @gmarull here. The rationale I use in general is:
- If the PR is rewriting the file from scratch then the copyright can be changed
- If the PR is changing significantly the content of the file the copyright can be added on top
- If the PR is not changing significantly the content of the file then the copyright is not touched

I had this same discussion with someone else a few days ago :)

Reply:
This file was created copying content from a file that did not have the `Copyright (c) 2022 Vestas Wind Systems A/S` copyright, so in theory it should have kept them as well. However now it's getting entirely re-written, so I think it's 100% Nordic owned content after the change. I understand what is copyrighted is the content.

Reply:
> file has been re-written from scratch, there's no re-used content

If that is the case, there should be two commits: one removing the existing file completely, and another one adding the new file.

Reply:
> If that is the case, there should be two commits: one removing the existing file completely, and another one adding the new file.

Please, only if that doesn't break bisectability.

Reply:
> > file has been re-written from scratch, there's no re-used content
> 
> If that is the case, there should be two commits: one removing the existing file completely, and another one adding the new file.

done, I have done the change in 2 steps, first delete old flags and add _new_ ones.